Dockerfiles/midstream/run_tests.py

```diff
@@ -4,45 +4,71 @@
 import json
 import unittest
 import subprocess
+import logging
 from os import path
 
 
+TEST_WORKING_DIR= os.getcwd()
+# global variables set
+OPERATOR_DIR=TEST_WORKING_DIR+"/operator_dir/"
+OPERATOR_WORK_DIR=TEST_WORKING_DIR+"/test_operator_work_dir/"
+WORK_DIR=TEST_WORKING_DIR+"/output/"
+PLAYBOOKS_DIR="/operator-test-playbooks/"
+ERROR_MESSAGE_PATH=TEST_WORKING_DIR+"/.errormessage"
+logging.basicConfig(filename=ERROR_MESSAGE_PATH)
+
 class RunMidstreamCVPTests():
 
     def setUp(self):
-        self.operator_dir = os.getenv('OPERATOR_DIR',
-                                      "/project/operator_dir/")
-        self.operator_work_dir = os.getenv('OPERATOR_WORK_DIR',
-                                           "/project/test_operator_work_dir/")
-        self.work_dir = os.getenv('WORK_DIR',
-                                  "/project/output/")
-        self.playbooks_dir = os.getenv('PLAYBOOKS_DIR',
-                                       "/project/operator-test-playbooks/")
+        # IMAGE_TO_TESTis an environment variable that takes 
+        # operator-bundle-image 
         self.image_to_test = os.getenv('IMAGE_TO_TEST')
-        self.umoci_bin_path = os.getenv('UMOCI_BIN_PATH',
-                                        '/usr/local/bin/umoci')
+        # VERBOSITY is verbosity of output log range varying from 1 to 4
+        self.verbosity = int(os.getenv('VERBOSITY', '1'))*"v"
+
+    def run_subprocess_command(self, exec_cmd):
+        print(exec_cmd)
+        env = os.environ.copy()
+        env["ANSIBLE_CONFIG"] = "/operator-test-playbook/Dockerfiles/midstream/"
+        env["ANSIBLE_LOCAL_TEMP"] = "/tmp/"
+
+        result = {}
+        process = subprocess.run(exec_cmd,
+                              stdout=subprocess.PIPE,
+                              stderr=subprocess.PIPE,
+                              shell=True,
+                              universal_newlines=True,
+                              env=env)
+        result["stdout"] = process.stdout
+        result["stderr"] = process.stderr
+        result["return_code"] = process.returncode
+        return result
 
     def run_extract_operator_bundle(self):
-        exec_cmd = "ansible-playbook -i localhost, -c local -v operator-test-playbooks/extract-operator-bundle.yml \
+        exec_cmd = "/usr/bin/ansible-playbook -i localhost, -c local -{verbosity} {playbook_dir}/extract-operator-bundle.yml \
```

Comment:
`ansible-playbook` is much shorter than `/usr/bin/ansible-playbook`

Reply:
This was introduced when ansible-playbook was not resolved by default.